test/testunusedvar.cpp

```diff
@@ -4839,6 +4839,23 @@ class TestUnusedVar : public TestFixture {
             "   H h;\n"
             "}");
         ASSERT_EQUALS("", errout.str());
+
+        // 2 translation units
```

Comment:
This is 1 translation unit. It's just 1 source file that includes some header(s).

If we forget the testrunner for a moment..

file1.cpp:

    #include "d.h"
    void f() {
        D d;
    }

d.cpp:

    #include "d.h"
    class D {
    public:
        D();
    };

d.cpp:

    #include "d.h"
    D::D() {}

When Cppcheck checks file1.cpp it cannot determine directly if D has a constructor body or not. That can only be determined when d.cpp is checked.

Reply:
My bad. Test updated.